test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const paths = require('paths');
+const termsAndConditionsContent = require('terms-and-conditions-page/content.en.json');
+
+Feature('Terms and Conditions Page');
```

Comment:
Do we really want to test all the content on the page? Maybe just do `I.see(pageTitle)` and then test all the links go to the correct urls? 
@paulgain

Reply:
OK. Whatever you think guys. In my opinion, I agree, maybe it's too much, but tbh, it's part of the ticket requirements, on the other hand, and it's not going to harm if we leave it.  But, as I said, I am happy to follow whatever you decide guys. @paulgain , your input?

Reply:
I agree, that's a complete waste of time. If we did that for every page the end-to-end tests would take a significant amount of time to run.

Reply:
end to end tests will take a significant amount of time, eventually, anyway. And then we will have to define our smoke-tests and run that shorter tests often instead. Having said that, I am happy to remove all that if you guys think so :)

Reply:
Yep, I agree the tests take a significant amount of time to run, lets not make the situation worse.

Reply:
okey dokey :)